pkg/scalers/authentication/authentication_helpers.go

```diff
@@ -80,6 +85,20 @@ func GetAuthConfigs(triggerMetadata, authParams map[string]string) (out *AuthMet
 			}
 			out.CustomAuthValue = authParams["customAuthValue"]
 			out.EnableCustomAuth = true
+		case OAuthType:
+			if out.EnableBasicAuth {
+				return nil, errors.New("both oauth and basic authentication can not be set")
+			}
+			if out.EnableBearerAuth {
+				return nil, errors.New("both oauth and bearer authentication can not be set")
+			}
+			out.EnableOAuth = true
+			out.OauthTokenURI = authParams["oauthTokenURI"]
+			if authParams["scope"] != "" {
```

Comment:
could you please trim the string first and change the format to accept **a comma separated list* of scopes?  To be consistent with the rest of the config.